app/frontend/styles/_tabs.scss

```diff
@@ -1,16 +1,111 @@
-.app-tabs {
-  margin-top: govuk-spacing(8);
+.app-tab-navigation {
```

Comment:
@steventux Can we rename this file `_tab-navigation.scss` a.) to match the namespace used by this class and b.) to avoid conflicts should we reintroduce `.govuk-tabs` and want to override some of its design in a `_tabs.scss` file.